app/page.tsx

```diff
@@ -311,8 +311,8 @@ export default function Home() {
                 paddingBottom: "30px",
               }}
             >
-              We leverage what emails already do... <br />
-              but using Regex & Zero Knowledge
+            We leverage what emails already do... <span className="hidden sm:inline"><br /></span>
```

Comment:
What is happening here?

Reply:
removing the line break on extra small screens but keeping it on screens small and above - for mobile responsiveness